### PR TITLE
Patch webprofiler to allow saving fields

### PR DESCRIPTION
**Pull request by mariano-dagostino**
#320

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -103,6 +103,9 @@
             },
             "drupal/core": {
               "d.o #3224254": "https://www.drupal.org/files/issues/2021-07-19/drupal-claro_dropdown-3224254-2.patch"
+            },
+            "drupal/devel": {
+              "d.o #43": "patches/419-Webprofiler-related-crash-in-core-9.4.2.patch"
```
Comment (amitaibu) [inline composer.json:108]: Let's follow other patches, and use the d.o URL, rather than have it locally

**Review by amitaibu — APPROVED** [review]
Thanks - one comment, and MWG

File: composer.json (hunk 1 above)
Comment (mariano-dagostino) [reply composer.json:108]: @amitaibu devel module does not uses drupal.org and issue tracking is done at https://gitlab.com/drupalspoons/devel

Comment (mariano-dagostino) [reply composer.json:108]: ah. I see your point...

Comment (amitaibu) [reply composer.json:108]: GitHub allows creating patch files by appending `.patch` to the PR - for example https://github.com/Gizra/drupal-starter/pull/321.patch . 
Gitlab likely has something similar

Comment (mariano-dagostino) [reply composer.json]: @amitaibu Yes, but there is no open PR for that issue, only a patch attached in a comment.

**Pull request merged**